var/spack/repos/builtin/packages/cdd/package.py

```diff
@@ -0,0 +1,47 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+
+from spack import *
+
+class Cdd(Package):
+    """The program cdd+ (cdd, respectively) is a C++ (ANSI C)
```

Comment:
Are we really enforcing a convention for package descriptions? Is that documented somewhere? I wouldn't want to receive differing requests from different reviewers. What about existing high-profile packages (GCC, HDF5, OpenMPI) -- have they been updated to be good examples? Why do we want package descriptions that start with incomplete English sentences?

So far I've been simply copying the blurbs from the packages' web sites. If that's how they advertise themselves, it's good enough for most use cases.